examples/patterns/navigation/subnav.html

```diff
@@ -0,0 +1,167 @@
+        </a>
+        <ul class="p-subnav u-hide" id="link-1-menu" aria-hidden="true">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav__toggle" role="menuitem" id="link-2">
+        <a href="#link-2-menu" class="p-subnav__toggle-link">
+          Link 2 <i class="p-subnav__toggle-indicator"></i>
+        </a>
+        <ul class="p-subnav u-hide" id="link-2-menu" aria-hidden="true">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 4</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 5</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav__toggle" role="menuitem" id="link-3">
+        <a href="#link-3-menu" class="p-subnav__toggle-link">
+          Link 3 <i class="p-subnav__toggle-indicator"></i>
+        </a>
+        <ul class="p-subnav u-hide" id="link-3-menu" aria-hidden="true">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </nav>
+</header>
+
+<script>
+var dropdowns = document.querySelectorAll('.p-subnav');
+var dropdownToggles = document.querySelectorAll('.p-subnav__toggle');
+var dropdownToggleLinks = document.querySelectorAll('.p-subnav__toggle-link');
+
+dropdownToggleLinks.forEach(function(dropdownToggleLink) {
+  dropdownToggleLink.addEventListener('click', function(event) {
+    event.preventDefault();
+  });
+});
+
+dropdownToggles.forEach(function(dropdownToggle) {
+  dropdownToggle.addEventListener('click', function(event) {
+    event.stopPropagation();
+
+    var clickedDropdown = this;
+
+    dropdownToggles.forEach(function(dropdownToggle) {
+      var dropdownContent = document.getElementById(dropdownToggle.id + "-menu");
+      if (dropdownToggle === clickedDropdown) {
+        if (dropdownToggle.classList.contains('is-active')) {
+          closeMenu(dropdownToggle, dropdownContent);
+        } else {
+          dropdownToggle.classList.add('is-active');
+          showElement(dropdownContent);
+
+          if (window.history.pushState) {
+            window.history.pushState(null, null, '#' + dropdownToggle.id);
```

Comment:
These are small drop downs so I wouldn't add the pushState

Reply:
Removed